src/components/ToolIcon.scss

```
@@ -83,3 +83,12 @@
     box-shadow: none;
   }
 }
+
+.ToolIcon__keybinding {
+  position: absolute;
+  bottom: 2px;
+  right: 2px;
```

```suggestion
  right: 4px;
```

I think it looks a bit better like this.. or maybe `3px`.. WDYT?

---

At the moment they look to tight to the border.

---

used 3px